<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignalgeneratorWidget</class>
 <widget class="QWidget" name="SignalgeneratorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1093</width>
    <height>594</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>197</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Generator</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Frequency:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="SIUnitEdit" name="frequency"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Level Control</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QDoubleSpinBox" name="levelSpin">
            <property name="suffix">
             <string>dbm</string>
            </property>
            <property name="minimum">
             <double>-50.000000000000000</double>
            </property>
            <property name="maximum">
             <double>0.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.250000000000000</double>
            </property>
            <property name="value">
             <double>-10.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="levelSlider">
            <property name="minimum">
             <number>-5000</number>
            </property>
            <property name="maximum">
             <number>0</number>
            </property>
            <property name="singleStep">
             <number>25</number>
            </property>
            <property name="pageStep">
             <number>500</number>
            </property>
            <property name="value">
             <number>-1000</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Enable</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QCheckBox" name="EnablePort1">
            <property name="text">
             <string>Port 1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="EnablePort2">
            <property name="text">
             <string>Port 2</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>196</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SIUnitEdit</class>
   <extends>QLineEdit</extends>
   <header>CustomWidgets/siunitedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
